src/components/directory/DirectoryPage.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+// import VisibilityOffIcon from '@material-ui/icons/VisibilityOff';
```

Comment:
Are these for future features?

Reply:
I was thinking for passwords, we could toggle to make it visible/masked. What do you think?

Reply:
Yuppp that is a good thing to add :D